scripts/iconform

```diff
@@ -663,10 +663,10 @@ def create_non_mip_output(variables, definitions, outputpath):
 # main
 #===================================================================================================
 def main(argv=None):
-   
+
     args = parseArgs(argv)
-    
-    print "\n" 
+
+    print "\n"
```

Comment:
This print statement and all others will have to be changed in order for the code to be Python 3 compatible.